### PR TITLE
Index for blog

### PR DESCRIPTION
**Pull request by tristinwells**

### CODE REVIEW

**Comment by tristinwells** [comment]
@troygrosfield

File: templates/index.html [hunk 1]
```diff
@@ -64,10 +64,15 @@ <h3 class="text-muted">Meet Tristin Wells</h3>
         
           <h4><a href="/about">About</a></h4>
           <p>Come see what I am all about!</p>
+          
           <p><iframe width ="100%" height = "300" scrolling="Yes" 
           src="https://moving.media/movies/5/paradise/embed"></iframe>
           
           </p>
+          <iframe width = "100%" height = "300" scrolling = "Yes"
```
Comment (troygrosfield) [inline templates/index.html:72]: don't put spaces between the "=".  This should instead be

```
<iframe width="100%" height="300" scrolling="Yes" ...
```

Comment (troygrosfield) [inline templates/index.html:72]: Once that change is make, everything else looks good.  Feel free to merge.

**Comment by troygrosfield** [comment]
Fixes #3.

File: templates/index.html [hunk 2]
```diff
@@ -64,10 +64,15 @@ <h3 class="text-muted">Meet Tristin Wells</h3>
         
           <h4><a href="/about">About</a></h4>
           <p>Come see what I am all about!</p>
-          <p><iframe width ="100%" height = "300" scrolling="Yes" 
+          
+          <p><iframe width="100%" height="300" scrolling="Yes" 
           src="https://moving.media/movies/5/paradise/embed"></iframe>
           
           </p>
+          <iframe width = "100%" height = "300" scrolling = "Yes"
```
Comment (troygrosfield) [inline templates/index.html:72]: this still needs to be fixed the same way it was fixed above.

**Pull request merged**